examples/CalcQt/CMakeLists.txt

```diff
@@ -1,36 +1,14 @@
 project(CalcQt)
 
-set(CALCQT_HEADERS src/CalculatorWidget.h)
-set(CALCQT_SOURCES src/CalcQt.cpp src/CalculatorWidget.cpp)
-include_directories(${CUKE_INCLUDE_DIRS} src)
+set(CALCQT_HEADERS )
 
-find_package(Qt5Core QUIET)
-find_package(Qt5Widgets QUIET)
-find_package(Qt5Test QUIET)
-
-if(${Qt5Core_FOUND} AND ${Qt5Widgets_FOUND} AND ${Qt5Test_FOUND})
-    set(CMAKE_INCLUDE_CURRENT_DIR ON)
-    set(CMAKE_AUTOMOC ON)
-    set(QT_LIBRARIES Qt5::Core Qt5::Widgets Qt5::Test)
-
-    add_library(libcalcqt src/CalculatorWidget.cpp ${CALCQT_HEADERS})
+if(QT_LIBRARIES)
+    add_library(libcalcqt src/CalculatorWidget.cpp)
     target_link_libraries(libcalcqt ${QT_LIBRARIES})
     
-    add_executable(calcqt ${CALCQT_SOURCES})
+    add_executable(calcqt src/CalcQt.cpp src/CalculatorWidget.h)
```

Comment:
I don't think you should be adding the `CalculatorWidget.h` header file to this executable: it's logically part of the `libcalcqt` library, not this.